core.fnl

```diff
@@ -74,6 +74,8 @@ Returns nil. This function causes side-effects.
           "
           (error "get-config can only be called after all modules have initialized")))
 
+(global spoon (or _G.spoon {}))
```

Comment:
I think this probably makes sense higher up, outside of `defaults`, maybe right after a lot of the imports.

Reply:
Moved up, added docs